packages/components/tests/dummy/app/controllers/components/pagination.js

```diff
@@ -9,4 +9,14 @@ export default class PaginationController extends Controller {
   toggleHighlight() {
     this.showHighlight = !this.showHighlight;
   }
+
+  @action
+  logClickedControl(control) {
+    console.log(`Control "${control}" clicked!`);
+  }
+
+  @action
+  handlePageChange(page) {
+    console.log(`Page changed to "${page}"!`);
```

Comment:
Do you want to leave these console logs in for now?

Reply:
no, it's on purpose: it's to have something to show to the consumers, that they have clicked a button and triggered an event/callback, when they play with one of the examples in the "how to use" or "showcase"